src/backend/aspen/workflows/align_gisaid/align.sh

```diff
@@ -59,7 +59,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/workflows/align_gisaid/save.py                 \
+entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/align_gisaid/save.py                 \
```

Comment:
can you fix the `\`

Reply:
do you want it to be in line with the previous `\`?